### PR TITLE
Stop operation script

### PR DESCRIPTION
**Pull request by JulianaCP-v2**

### CODE REVIEW

File: operator_toolkit/stop_operation.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    on that site, taking in consideration the operation type.
+
+    :param str site: site id to process.
+    :param str operation_type: type of service operation to process.
+    :param str stop: kind of stop operation to process
+    :return: returns the result of the operation.
+    :rtype: str
+    """
+    stop_url = "modes/stop" if stop == "mode" else "stop"
+    url = f'http://{operation_type}-{site}.axiomcloud.io/{stop_url}'
```
Comment (jacksboo) [inline operator_toolkit/stop_operation.py:26]:
```suggestion
    url = f'https://{operation_type}-{site}.axiomcloud.io/{stop_url}'
```
I think the request should use `https`

**Review by jacksboo — COMMENTED** [review]
Looks good to me, with Only a single comment.

File: config.yml [hunk 2]
```diff
@@ -1,4 +1,8 @@
 sites:
+  ax-test:
+    controller: microthermo
+    organization: axiom
+
```
Comment (samhollenbach) [inline config.yml:5]: Let's make sure this change is propagated to S3 as well.

**Review by samhollenbach — APPROVED** [review]
Looks good

**Pull request merged**